Song.Tests/SongWriterTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song.Tests
+{
+    public class SongWriterTests
```

Comment:
Global:
- Test names tend to be very long. It's not readable, when you don't separate different words or parts in test using `_` or similar. You can name a test like this: `MethodUnderTest_GivenPrecondition_WhenInput_Expectation`